src/assets/styles/antd-customize/generic.less

```diff
@@ -140,6 +140,12 @@
     color: white !important;
   }
 
+  .ant-table-empty{
```

Comment:
The strange white overlay on the fixed right column. Only happen when table is empty and has `fixed: right`.

<img width="480" alt="image" src="https://github.com/longhorn/longhorn-ui/assets/5744158/3069de27-401b-4a31-8d3e-13e46bd281a7">

After fix
<img width="452" alt="Screenshot 2024-06-06 at 8 40 44 PM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/7251b20b-ec49-4c80-8d67-c66cf2df4684">